### ISSUE TITLE
xPRO CI can't sync courseruns and grades

### ISSUE THREAD

**Issue by pdpinch** [mitodl/ol-infrastructure#1239]
# User Story

As part of testing the Olive release of open edX, we'd like to test:
1. syncing course metadata from open edX to xPRO
2. syncing grades from open edX to xPRO

# Description/Context

Course syncing is failing with a 401 error. I suspect the necessary privileged user account isn't set up. 

https://sentry.io/organizations/mit-office-of-digital-learning/issues/1380117045/?environment=ci&project=1413655&query=is%3Aunresolved&referrer=issue-stream

On RC and prod there is an odl devops user that I presume is the one used for this purpose.

# Acceptance Criteria
- [ ] set up the privileged account that can request course metadata for all courses and grades from all courses

**Comment by shaidar** [comment]
@pdpinch Can u try now or let me know how I can test this and with what course.

**Comment by pdpinch** [comment]
I'm still getting a 401 error when I run the management command to sync courses. 

```
~ $ python manage.py sync_courseruns --run course-v1:XPRO+TESTING+TESTR0
[2022-12-06 21:18:04] ERROR 18 [courses.utils] utils.py:260 - [a187e460-69dc-440e-be56-996721048d07] - 401 Client Error: Unauthorized for url: https://courses-ci.xpro.mit.edu/api/courses/v1/courses/course-v1:XPRO+TESTING+TESTR0/?username=xpro-service-worker-api: course-v1:XPRO+TESTING+TESTR0
Sync complete: 0 updated, 1 failures
Sentry is attempting to send 2 pending error messages
Waiting up to 2 seconds
Press Ctrl-C to quit
```
```


`python manage.py sync_courseruns`

**Comment by pdpinch** [comment]
Thanks! Working now:

```
% heroku run --app xpro-ci bash
 ›   Warning: heroku update available from 7.66.4 to 7.67.1.
Running bash on ⬢ xpro-ci... up, run.3225 (Standard-1X)
~ $ python manage.py sync_grades_and_certificates --run course-v1:XPRO+TESTING+TESTR0
Processed user [REDACTED] ([REDACTED]) in course run course-v1:XPRO+TESTING+TESTR0. Result - Grade: already exists (passed: True), Certificate: already exists
Processed user [REDACTED] ([REDACTED]) in course run course-v1:XPRO+TESTING+TESTR0. Result - Grade: already exists (passed: False), Certificate: ignored
~ $ python manage.py sync_courseruns --run course-v1:XPRO+TESTING+TESTR0
[2022-12-07 12:56:44] INFO 15 [courses.utils] utils.py:282 - [18205559-bdfe-4ca6-8562-8e1a0008a19c] - Updated course run: course-v1:XPRO+TESTING+TESTR0
Sync complete: 1 updated, 0 failures
```
